src/components/BorrowerProfile/CountryInfo.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<section>
+		<h2 class="tw-text-h2">
+			{{ countryName }} at a glance
+		</h2>
+		<div class="tw-flex lg:tw-mb-3">
+			<div class="tw-flex-auto">
+				<h2 class="tw-text-h2 tw-block tw-m-0">
```

Comment:
Minor, but I think this and the "loans currently fundraising" below would be better marked up as a paragraph tag, with 2 spans inside for styling. This would let a screenreader announce them together instead of as separate sentences.

Something like
```
<div class="tw-flex-auto">
		<p>
			<span class="tw-block tw-text-h2">$2500</span>
			<span class="tw-block tw-text-h4 tw-bg-gray-500">average annual income (USD)</span>
		</p>
</div>
```

Reply:
+1